src/main/kotlin/io/github/rybalkinsd/kohttp/client/OkHttpClientExt.kt

```diff
@@ -1,11 +1,22 @@
 package io.github.rybalkinsd.kohttp.client
 
+import okhttp3.Interceptor
 import okhttp3.OkHttpClient
 
 /**
  *
  * @since 0.5.0
  * @author sergey
  */
-fun OkHttpClient.fork(block: ForkClientBuilder.() -> Unit) : OkHttpClient =
-    ClientBuilderImpl(this).apply(block).build()
+fun OkHttpClient.fork(block: ForkClientBuilder.() -> Unit): OkHttpClient =
+        ClientBuilderImpl(this).apply(block).build()
+
+
+class InterceptorsDsl(private val interceptors: MutableList<Interceptor> = mutableListOf()) {
```

Comment:
need to add test where
`base` client has an interceptor and `fork` client adds one more